packages/testing-helpers/src/helpers.js

```diff
@@ -11,12 +11,12 @@ let defineCECounter = 0;
  * const el = fixture(`<${tag}></${tag}>`);
  * // test el
  *
- * @param {typeof HTMLElement | function} klass Class which extends HTMLElement
+ * @template {HTMLElement} T
+ * @param {import("@open-wc/dedupe-mixin").Constructor<T>} klass Class which extends HTMLElement
```

Comment:
wouldn't that require a hard dependency on `@open-wc/dedupe-mixin`?

Reply:
LitElement also exports this type

Reply:
also, that hard dependency is introduced by scopedelementstestwrapper